examples/node-agent-live/index.js

```
@@ -0,0 +1,110 @@
+const { writeFile, appendFile } = require("fs/promises");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)